test/module/irohad/consensus/yac/yac_mocks.hpp

```diff
@@ -18,198 +18,189 @@
 #ifndef IROHA_YAC_MOCKS_HPP
 #define IROHA_YAC_MOCKS_HPP
 
-#include <gtest/gtest.h>
 #include <gmock/gmock.h>
 
+#include "consensus/yac/cluster_order.hpp"
+#include "consensus/yac/messages.hpp"
 #include "consensus/yac/yac.hpp"
 #include "consensus/yac/yac_gate.hpp"
-#include "consensus/yac/cluster_order.hpp"
 #include "consensus/yac/yac_hash_provider.hpp"
 #include "consensus/yac/yac_peer_orderer.hpp"
-#include "consensus/yac/messages.hpp"
 
-using namespace iroha::consensus::yac;
-using iroha::model::Peer;
+namespace iroha {
+  namespace consensus {
+    namespace yac {
+      model::Peer mk_peer(std::string address) {
+        model::Peer peer;
+        peer.address = address;
+        return peer;
+      }
 
-Peer mk_peer(std::string address) {
-  Peer peer;
-  peer.address = address;
-  return peer;
-}
+      VoteMessage create_vote(YacHash hash, std::string sign) {
+        VoteMessage vote;
+        vote.hash = hash;
+        std::copy(sign.begin(), sign.end(), vote.signature.pubkey.begin());
+        return vote;
+      }
 
-VoteMessage create_vote(YacHash hash, std::string sign) {
-  VoteMessage vote;
-  vote.hash = hash;
-  std::copy(sign.begin(), sign.end(),
-            vote.signature.pubkey.begin());
-  return vote;
-}
+      class MockYacCryptoProvider : public YacCryptoProvider {
+       public:
+        MOCK_METHOD1(verify, bool(CommitMessage));
+        MOCK_METHOD1(verify, bool(RejectMessage));
+        MOCK_METHOD1(verify, bool(VoteMessage));
 
-/**
- * Mock for yac crypto provider
- */
-class CryptoProviderMock : public YacCryptoProvider {
- public:
-  MOCK_METHOD1(verify, bool(CommitMessage));
-  MOCK_METHOD1(verify, bool(RejectMessage));
-  MOCK_METHOD1(verify, bool(VoteMessage));
+        VoteMessage getVote(YacHash hash) override {
+          VoteMessage vote;
+          vote.hash = hash;
+          return vote;
+        }
 
-  VoteMessage getVote(YacHash hash) override {
-    VoteMessage vote;
-    vote.hash = hash;
-    return vote;
-  };
+        MockYacCryptoProvider() = default;
 
-  CryptoProviderMock() {};
+        MockYacCryptoProvider(const MockYacCryptoProvider &) {}
 
-  CryptoProviderMock(const CryptoProviderMock &) {};
+        MockYacCryptoProvider &operator=(const MockYacCryptoProvider &) {
+          return *this;
+        }
+      };
 
-  CryptoProviderMock &operator=(const CryptoProviderMock &) { return *this; };
-};
+      class MockTimer : public Timer {
+       public:
+        void invokeAfterDelay(uint64_t millis,
+                              std::function<void()> handler) override {
+          handler();
+        }
 
-/**
- * Mock for timer
- */
-class FakeTimer : public Timer {
- public:
-  void invokeAfterDelay(uint64_t millis,
-                        std::function<void()> handler) override {
-    handler();
-  };
+        MOCK_METHOD0(deny, void());
 
-  MOCK_METHOD0(deny, void());
+        MockTimer() {}
```

Comment:
We can review all constructors and assignment operators of mocks to leave only required ones, and add comments why they are required